examples/gcp/import/provider.tf

```diff
@@ -0,0 +1,30 @@
+# Specify the provider and access details
+provider "netapp-gcp" {
+  # numerical project number (not project ID)
```

Comment:
I thought we changed this, to accept both.   Let's double check.

Reply:
Yes, this was changed in e2f3cd2f415e650791972a01e195ffd7f33ae17d

Reply:
Changed it for all examples.